docs/snippets/web-components/button-group-story.ts.mdx

```diff
@@ -0,0 +1,33 @@
+```ts
+// demo-button-group.stories.ts
+import type { Meta, StoryObj } from '@storybook/web-components';
+
+import { ButtonGroup, ButtonGroupProps } from './demo-button-group';
+
+//👇 Imports the Button stories
+import * as ButtonStories from './demo-button.stories';
```

Comment:
@kylegach, when you're able, can you update the imports across your pr to the following structure so it matches the rest of the snippets and examples we provide once Storybook is added to a Web Components' project:

```ts
// ButtonGroup.stories.ts

import type { Meta, StoryObj } from '@storybook/web-components';

import type { ButtonGroupProps } from './ButtonGroup';
import { ButtonGroup } from './ButtonGroup';

const meta: Meta<ButtonGroupProps> = {
  title: 'ButtonGroup',
  render: (args) => ButtonGroup(args),
};

// Remainder implementation
```

The main reasons for this are that as of the latest changes, we're not defining the examples as `demo-*` but instead calling them by their actual names. Also, removing the comment regarding the title was intentional, as, during my testing, if we don't name the story, Storybook throws an error.